enumer.go

```diff
@@ -37,6 +41,7 @@ func %[1]sStrings() []string {
 `
 
 // Arguments to format are:
+//
```

Comment:
these newlines are generated by `gofmt` or something. I don't think they should be here, but I also don't know what the correct alternative to get `gofmt` to stop doing this is

Reply:
I think this is standard `gofmt` behavior because this line `//	[1]: type name` starts with a tab character. It adds the newline to separate between the comment on line 43 and what it thinks is a list

Reply:
maybe I can just put them on a single line? I think that's what I'll do.